cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/executor/Executor.java

```diff
@@ -739,12 +740,8 @@ public synchronized void setGeneratingProposalsForExecution(String uuid, Supplie
     if (currentExecutorState != NO_TASK_IN_PROGRESS) {
       throw new OngoingExecutionException(String.format("Cannot generate proposals while the executor is in %s state.", currentExecutorState));
     }
-    if (uuid == null) {
-      throw new IllegalArgumentException("UUID of the execution cannot be null.");
-    }
-    if (reasonSupplier == null) {
-      throw new IllegalArgumentException("Reason supplier cannot be null.");
-    }
+    validateNotNull(uuid, "UUID of the execution cannot be null.");
```

Comment:
Same issue: `_uuid  = validateNotNull(uuid, "UUID of the execution cannot be null.");`